### DISCUSSION TITLE
[Question]: How to provide custom instructions similar to chatgpt UI

### DISCUSSION THREAD

**Discussion by rtmlp** [danny-avila/LibreChat#4593] [Q&A]
### What is your question?

Hi, 

In chatgpt's webui, there are two questions for `Custom Instructions`

1. What would you like ChatGPT to know about you to provide better responses?
2. How would you like ChatGPT to respond?

Where can I provide my responses to these questions while sending request to openai. 

### More Details

In presets, when I set the custom instructions, am I setting it for 1st question or 2nd question. Is there any way to set these for all the calls ?

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
ChatGPT is combining the answers to those 2 questions into 1 formatted prompt--it's up to you to format the instructions as desired.
